bindings/nodejs/code/stronghold_storage.ts

```diff
@@ -0,0 +1,110 @@
+import { Buffer } from 'buffer';
+import { NapiStronghold, NapiDID, NapiKeyLocation, NapiChainState, NapiIdentityState, NapiGeneration } from '../index.js'
+import { DID, KeyLocation, Generation, DIDLease, Signature, ChainState, IdentityState, Storage } from "../../../wasm/node/identity_wasm.js";
```

Comment:
Skeptical about this dependency. If the Wasm bindings change, without these bindings being updated things might break. Ideally the TS types and JS functions get created from the Rust implementation in these bindings.

Reply:
It uses serialization to convert the values so as long as the JSON representations remain compatible (which should be true for non-major semver changes, even when adding new fields) it should always work.